### PR TITLE
Task/DES-1083 - Finish Field Research pipeline and Fix template bugs

### PR DESCRIPTION
**Pull request by Strmiska**

### CODE REVIEW

File: designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js [hunk 1]
```diff
@@ -53,6 +55,15 @@ class PipelineAuthorsCtrl {
                 }
             };
             this.verifyAuthors(this.experiment.value.authors);    
+            if (this.projType === 'experimental') {
+                this.placeholder = 'Experiment';
+            } else if (this.projType === 'simulation') {
+                this.placeholder = 'Simulation';
+            } else if (this.projType === 'hybrid_simulation') {
+                this.placeholder = 'Hybrid Simulation';
+            } else if (this.projType === 'field_recon') {
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js]: What about `projType == 'other'`?

File: designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js [hunk 2]
```diff
@@ -84,6 +95,13 @@ class PipelineAuthorsCtrl {
                 experiment: this.experiment,
                 selectedListings: this.selectedListings,
             }, {reload: true});
+        } else if (this.projType === 'field_recon') {
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js]: `projType = 'other'`?

File: designsafe/static/scripts/data-depot/components/projects/pipeline-field-recon/pipeline-field-recon.component.html [hunk 3]
```diff
@@ -76,48 +76,78 @@ <h3 style="text-align: center; color: orange;">
...
-                <span ng-if="author.guest">{{ author.lname }}, {{ author.fname }};</span>
-            </span>
-            <span class="authors-end">__</span>
-            <!-- older author list -->
-            <div ng-repeat="author in $ctrl.mission.value.authors" ng-if="author.length">
-                <ds-user username="author" format="lname">{{ author }}</ds-user>
+            <div style="display:inline-block; width:20%;">
+                DOI 
+                <button class="btn btn-info btn-sm"
+                        data-ng-click="$ctrl.showCitation(missino)">
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-field-recon/pipeline-field-recon.component.html]: should this be
```suggestion
                        data-ng-click="$ctrl.showCitation($ctrl.mission)">
```
?

File: designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.js [hunk 4]
```diff
@@ -60,6 +60,15 @@ class PipelineLicensesCtrl {
...
             this.projType = this.project.value.projectType;
+            if (this.projType === 'experimental') {
+                this.placeholder = 'Experiment';
+            } else if (this.projType === 'simulation') {
+                this.placeholder = 'Simulation';
+            } else if (this.projType === 'hybrid_simulation') {
+                this.placeholder = 'Hybrid Simulation';
+            } else if (this.projType === 'field_recon') {
+                this.placeholder = 'Mission';
+            }
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.js]: `projType == 'other'`?

File: designsafe/static/scripts/data-depot/components/projects/pipeline-project/pipeline-project.component.js [hunk 5]
```diff
@@ -34,12 +34,23 @@ class PipelineProjectCtrl {
...
             });
         } else {
             this.projType = this.project.value.projectType;
+            if (this.projType === 'experimental') {
+                this.placeholder = 'Experiment';
+            } else if (this.projType === 'simulation') {
+                this.placeholder = 'Simulation';
+            } else if (this.projType === 'hybrid_simulation') {
+                this.placeholder = 'Hybrid Simulation';
+            } else if (this.projType === 'field_recon') {
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-project/pipeline-project.component.js]: `projType == 'other'`?

File: designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js (hunk 1 above)
Comment (Strmiska) [reply designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js]: `projType == 'other'` projects handle authors differently than other projects. We use `pipeline-team.component.js` in this case.

File: designsafe/static/scripts/data-depot/components/projects/pipeline-field-recon/pipeline-field-recon.component.html (hunk 3 above)
Comment (Strmiska) [reply designsafe/static/scripts/data-depot/components/projects/pipeline-field-recon/pipeline-field-recon.component.html]: yep!

File: designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.js (hunk 4 above)
Comment (Strmiska) [reply designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.js]: pipeline is a little different for 'other' so the placeholder is not necessary

File: designsafe/static/scripts/data-depot/components/projects/pipeline-categories/pipeline-categories-field-recon.component.html [hunk 6]
```diff
@@ -73,7 +73,7 @@ <h3 style="text-align: center; color: orange;">
     </div>
     <!-- Categories Start -->
     <!-- Reports -->
-    <div ng-repeat="report in $ctrl.browser.project.report_set | orderBy: 'value.title'"
+    <div ng-repeat="report in $ctrl.project.report_set | orderBy: 'value.title'"
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-categories/pipeline-categories-field-recon.component.html]: Does all of field-recon replace `browser.project` with `project`?

Comment (Strmiska) [reply designsafe/static/scripts/data-depot/components/projects/pipeline-categories/pipeline-categories-field-recon.component.html]: Project metadata for all project types is assigned to `this.project` when inside the pipeline. However, I noticed that the DataBrowserService was still being imported and assigned in the controller. It's not being used here, so I should just take it out.

Comment (rstijerina) [reply designsafe/static/scripts/data-depot/components/projects/pipeline-categories/pipeline-categories-field-recon.component.html]: There are other areas where `browser.project` is being used, should those be using `project` as well?

**Pull request merged**